Build fails on Linux: main.h:14:21: fatal error: tox/tox.h: No such file or directory

I just cloned uTox for the first time and tried to build it, but:

- Instructions in INSTALL.md and at https://wiki.tox.im/UTox#Compiling are contradictory
- Anyway both fail with the same error:

```
nico@nico:~/src$ git clone https://github.com/notsecure/uTox.git
Cloning into 'uTox'...
remote: Counting objects: 2869, done.
remote: Compressing objects: 100% (45/45), done.
remote: Total 2869 (delta 25), reused 12 (delta 4)
Receiving objects: 100% (2869/2869), 1.60 MiB | 637.00 KiB/s, done.
Resolving deltas: 100% (1962/1962), done.
Checking connectivity... done.
nico@nico:~/src$ cd uTox/
nico@nico:~/src/uTox$ sudo apt-get install libv4l-dev libopenal-dev libfreetype6-dev libdbus-1-dev libxrender-dev libfontconfig1-dev libxext-dev
[sudo] password for nico: 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
libxext-dev is already the newest version.
libxext-dev set to manually installed.
libxrender-dev is already the newest version.
libxrender-dev set to manually installed.
libdbus-1-dev is already the newest version.
libdbus-1-dev set to manually installed.
libfontconfig1-dev is already the newest version.
libfontconfig1-dev set to manually installed.
libfreetype6-dev is already the newest version.
The following extra packages will be installed:
  libv4l2rds0
The following NEW packages will be installed:
  libopenal-dev libv4l-dev libv4l2rds0
0 upgraded, 3 newly installed, 0 to remove and 64 not upgraded.
Need to get 141 kB of archives.
After this operation, 710 kB of additional disk space will be used.
Do you want to continue? [Y/n] 
Get:1 http://archive.ubuntu.com/ubuntu/ trusty/main libv4l2rds0 amd64 1.0.1-1 [15.9 kB]
Get:2 http://archive.ubuntu.com/ubuntu/ trusty/universe libopenal-dev amd64 1:1.14-4ubuntu1 [23.1 kB]
Get:3 http://archive.ubuntu.com/ubuntu/ trusty/main libv4l-dev amd64 1.0.1-1 [102 kB]
Fetched 141 kB in 3s (41.3 kB/s)     
Selecting previously unselected package libv4l2rds0:amd64.
(Reading database ... 423335 files and directories currently installed.)
Preparing to unpack .../libv4l2rds0_1.0.1-1_amd64.deb ...
Unpacking libv4l2rds0:amd64 (1.0.1-1) ...
Selecting previously unselected package libopenal-dev:amd64.
Preparing to unpack .../libopenal-dev_1%3a1.14-4ubuntu1_amd64.deb ...
Unpacking libopenal-dev:amd64 (1:1.14-4ubuntu1) ...
Selecting previously unselected package libv4l-dev:amd64.
Preparing to unpack .../libv4l-dev_1.0.1-1_amd64.deb ...
Unpacking libv4l-dev:amd64 (1.0.1-1) ...
Setting up libv4l2rds0:amd64 (1.0.1-1) ...
Setting up libopenal-dev:amd64 (1:1.14-4ubuntu1) ...
Setting up libv4l-dev:amd64 (1.0.1-1) ...
Processing triggers for libc-bin (2.19-0ubuntu6.3) ...
nico@nico:~/src/uTox$ make all
Package libtoxav was not found in the pkg-config search path.
Perhaps you should add the directory containing `libtoxav.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libtoxav' found
Package libtoxcore was not found in the pkg-config search path.
Perhaps you should add the directory containing `libtoxcore.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libtoxcore' found
Package vpx was not found in the pkg-config search path.
Perhaps you should add the directory containing `vpx.pc'
to the PKG_CONFIG_PATH environment variable
No package 'vpx' found
  CC    button.o
In file included from button.c:1:0:
main.h:14:21: fatal error: tox/tox.h: No such file or directory
 #include <tox/tox.h>
                     ^
compilation terminated.
make: *** [button.o] Error 1
```

--- a/docs/INSTALL.md
+++ b/docs/INSTALL.md
@@ -4,6 +4,8 @@ The following will help you get μTox installed on your computron/toaster/carrie
 get you running ASAP. And mostly deals with precomplied binarys. If you're looking to contribute (you're the real MVP) 
 you're probably looking for [Building](BUILD.md). 
 
+For any and all of the following, you'll need to have [toxcore](https://github.com/irungentoo/toxcore) installed first.
+
 - [Unix-like](#unix)
 - [OS X](#osx)
 - [Windows](#windows)

